src/researchhub_comment/related_models/rh_comment_model.py

```diff
@@ -0,0 +1,55 @@
+from django.db.models import CharField, FileField, TextField, ForeignKey, CASCADE, SET_NULL
+
+from discussion.reaction_models import AbstractGenericReactionModel
+from researchhub_comment.constants.rh_comment_content_types import QUILL_EDITOR, RH_COMMENT_CONTENT_TYPES
+from researchhub_comment.related_models.rh_comment_thread_model import RhCommentThreadModel
+from utils.models import DefaultAuthenticatedModel
+
+class RhCommentModel(AbstractGenericReactionModel, DefaultAuthenticatedModel):
+    # auth
+    created_by = ForeignKey(
+        "user.User",
+        auto_created=True,
+        on_delete=CASCADE,
+        related_name="created_rh_comments",
+    )
+    updated_by = ForeignKey(
+        "user.User",
+        auto_created=True,
+        on_delete=CASCADE,
+        related_name="updated_rh_comments",
+    )
```

Comment:
Does `DefaultAuthenticatedModel` come with these fields?

Reply:
We need to re-specify to form related_name for traversal reasons